### PR TITLE
Add recipe for google-auth-oauthlib

### PR DESCRIPTION
**Pull request by parthea**
For https://pypi.python.org/pypi/google-auth-oauthlib/0.1.0

This is waiting on conda-forge/staged-recipes#3099. Nothing to do here until conda-forge/staged-recipes#3099 is ready.

### CODE REVIEW

**Comment by parthea** [comment]
All checks passed. This is ready for review.

cc @jakirkham

File: recipes/google-auth-oauthlib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  summary: 'Google Authentication Library, oauthlib integration with google-auth'
+  description: This library provides oauthlib integration with google-auth.
+  doc_url: https://google-auth.readthedocs.io/en/latest/
+  dev_url: https://github.com/GoogleCloudPlatform/google-auth-library-python
+
+extra:
+  recipe-maintainers:
+    - parthea
+    - jreback
+    - tswast
```
Comment (jakirkham) [inline recipes/google-auth-oauthlib/meta.yaml]: Are both of you ok being maintainers here, @jreback and @tswast?

Comment (jreback) [inline recipes/google-auth-oauthlib/meta.yaml]: ok

Comment (tswast) [inline recipes/google-auth-oauthlib/meta.yaml]: yes

**Comment by parthea** [comment]
@jakirkham Please review at your earliest convenience

**Comment by jakirkham** [comment]
Thanks @parthea.

**Pull request merged**